.travis.yml

```diff
@@ -41,11 +41,16 @@ cache:
     - $HOME/.gimme
 
 services:
-  - mysql
   - memcached
   - redis
 
 before_install:
+  - wget https://dev.mysql.com/get/mysql-apt-config_0.8.15-1_all.deb
```

Comment:
Maybe we want an env var for `mysql-apt-config_0.8.15-1_all.deb`. It's referenced twice and that might help with bumping the version up in the future.

Reply:
I kinda like that it's not a variable tbh, it's less likely to be accidentally overwritten. I came up with a middle ground approach (and I also applied to elasticsearch downloading, which had the same issue) : f53b71a22eab2e03645600d12a646e2a0315ee66